app/grandchallenge/reader_studies/views.py

```diff
@@ -1438,3 +1441,71 @@ def form_valid(self, form):
             instance.save()
 
         return HttpResponseRedirect(self.get_success_url())
+
+
+class DisplaySetPDFReport(
+    LoginRequiredMixin, ObjectPermissionRequiredMixin, View
+):
+    template_name = "reader_studies/readerstudy_display_set_pdf_report.html"
+    permission_required = "reader_studies.change_readerstudy"
+    raise_exception = True
+
+    def get_permission_object(self):
+        return self.reader_study
+
+    @cached_property
+    def reader_study(self):
+        return get_object_or_404(ReaderStudy, slug=self.kwargs["slug"])
+
+    @cached_property
+    def display_set(self):
+        return get_object_or_404(DisplaySet, pk=self.kwargs["pk"])
+
+    @cached_property
+    def user(self):
+        return get_object_or_404(
+            get_user_model(), username=self.kwargs["username"]
+        )
+
+    def get(self, request, *args, **kwargs):
+        # Create a Django response object, and specify content_type as pdf
+        response = HttpResponse(content_type="application/pdf")
+        filename = "Report_" + str(self.display_set.pk) + ".pdf"
+        response["Content-Disposition"] = f"attachment; filename={filename}"
+
+        # find the template and render it.
+        template = get_template(self.template_name)
+        html = template.render(self.get_context_data())
+
+        # create a pdf
+        pisa_status = pisa.CreatePDF(html, dest=response)
+
+        if pisa_status.err:
+            return HttpResponseServerError(
+                "We encountered some errors <pre>" + html + "</pre>"
+            )
+
+        return response
+
+    def get_context_data(self):
+        answer_dict = {
+            answer.question.question_text: answer.answer_text
+            for answer in self.display_set.answers.select_related(
+                "creator", "question", "answer_image"
+            )
+            .filter(
+                Q(creator=self.user),
+                Q(is_ground_truth=False),
+                ~Q(question__answer_type__in=AnswerType.annotation_types()),
+                Q(answer_image__isnull=True),
+            )
```

Comment:
Better not to use Q objects:

```suggestion
            .filter(
                creator=self.user,
                is_ground_truth=False,
                answer_image__isnull=True,
            )
            .exclude(
                question__answer_type__in=AnswerType.annotation_types(),
            )
```